easybuild/easyconfigs/l/libwebp/libwebp-1.0.2-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'libwebp'
+version = '1.0.2'
+
+homepage = 'https://developers.google.com/speed/webp/'
+description = """WebP is a modern image format that provides superior
+lossless and lossy compression for images on the web. Using WebP,
+webmasters and web developers can create smaller, richer images that
+make the web faster."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['https://storage.googleapis.com/downloads.webmproject.org/releases/webp']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['3d47b48c40ed6476e8047b2ddb81d93835e0ca1b8d3e8c679afbb3004dd564b1']
+
+builddependencies = [('binutils', '2.30')]
+
+dependencies = [
+    ('libjpeg-turbo', '2.0.0'),
+    ('libpng', '1.6.34'),
+    ('LibTIFF', '4.0.9'),
+    ('giflib', '5.1.4'),
+]
+
+sanity_check_paths = {
+    'files': [],
```

Comment:
@akesandgren The existing `libwebp` easyconfig has a significantly more specific `sanity_check_paths`...

Reply:
Hmm, I apparently missed that there was one.
I'll update this one.